helper/schema/resource_data.go

```diff
@@ -213,9 +212,11 @@ func (d *ResourceData) State() *terraform.InstanceState {
 		}
 
 		raw := d.get([]string{k}, source)
-		rawMap[k] = raw.Value
-		if raw.ValueProcessed != nil {
-			rawMap[k] = raw.ValueProcessed
+		if raw.Exists && !raw.Computed {
```

Comment:
Why would you not save computed values to the state? How can you then persist info that is only computed during the create but not again during a refresh or update?